packages/webcert/src/feature/certificate/Inputs/UeSickLeavePeriod/DateRangePicker.tsx

```diff
@@ -263,12 +287,18 @@ const DateRangePicker: React.FC<Props> = ({
   const resetValidation = () => {
     setValidation({
       invalidDatePeriod: false,
+      notCompleteDate: false,
       validationErrors: [],
     })
   }
 
-  const getShouldDisplayValidationErrorOutline = () => {
-    return validations.invalidDatePeriod || hasOverlap || hasValidationError
+  const getShouldDisplayValidationErrorOutline = (id: string) => {
+    let hasNotCompleteDateValidation = false
+    if (validations.notCompleteDate) {
```

Comment:
Is this check really necessary? If I'm not mistaken you could replace the `if` statement with
```typescript
hasNotCompleteDateValidation =
        validations.validationErrors.some((v) => v.type === 'NOT_COMPLETE_DATE' && v.id.includes(id))
```